homeassistant/components/ios/__init__.py

```diff
@@ -160,6 +165,13 @@
 def devices_with_push(hass):
     """Return a dictionary of push enabled targets."""
     targets = {}
+
+    for device in async_devices_with_key(hass, DOMAIN, ATTR_PUSH_ID):
```

Comment:
The word device is ambiguous as we also have device registry. We should refer to it as registration.

Reply:
Removed for a later PR